frontend_lib/src/component/Timeline/Timeline.styl

```diff
@@ -104,6 +119,3 @@ columnTimeline()
     &__messagelist
       &__version__date
         font-size 15px
-    &__texteditor__advancedtext__btn
```

Comment:
keep that for now, if you think it is not a good idea to disable it on mobile, you can create a new issue about it

Reply:
Done.